File: src/main/kotlin/xyz/spotifynutzer/utils/labymod/BalanceType.java
package xyz.spotifynutzer.utils.labymod;

/**
 * @author kxmpetentes
 * Website: kxmpetentes.de
 * GitHub: git.kxmpetentes.de
 * Erstellt am: 07.03.2021 um 10:13
 */

public enum BalanceType {

    CASH("cash"),
    BANK("bank");

    private final String key;

    BalanceType(String key) {
        this.key = key;
    }

    public String getKey() {
        return this.key;
    }
}

File: src/main/kotlin/xyz/spotifynutzer/utils/labymod/BalanceType.kt
package xyz.spotifynutzer.utils.labymod;

/**
 * @author kxmpetentes
 * Website: kxmpetentes.de
 * GitHub: git.kxmpetentes.de
 * Erstellt am: 07.03.2021 um 10:13
 */

enum class BalanceType(private val key: String) {

    CASH("cash"),
    BANK("bank");

    fun getKey(): String {
        return key
    }
}
